### PR TITLE
use PORTAL_DOMAIN as COOKIE_DOMAIN

### PR DESCRIPTION
**Pull request by kwypchlo**
PORTAL_DOMAIN contains a correct domain that should also be used as COOKIE_DOMAIN

cc @ro-tex

### CODE REVIEW

**Review by peterjan — APPROVED** [review]
This is fine but I wonder whether the fix should have been to replace `COOKIE_DOMAIN` with `PORTAL_DOMAIN` instead, IMO that makes more sense as it removes complexity. This could then be documented in code when we create a cookie domain variable, e.g:
```
// use the portal domain as the cookie domain blabla
const cookieDomain = process.env.PORTAL_DOMAIN;
```

**Comment by kwypchlo** [comment]
> This is fine but I wonder whether the fix should have been to replace `COOKIE_DOMAIN` with `PORTAL_DOMAIN` instead, IMO that makes more sense as it removes complexity. This could then be documented in code when we create a cookie domain variable, e.g:
> 
> ```
> // use the portal domain as the cookie domain blabla
> const cookieDomain = process.env.PORTAL_DOMAIN;
> ```

I agree, we should make this change in https://github.com/SkynetLabs/skynet-accounts but that's just a quick fix that will work until we can do that properly.

File: docker-compose.accounts.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ services:
     environment:
       - ACCOUNTS_EMAIL_URI=${ACCOUNTS_EMAIL_URI}
       - ACCOUNTS_JWKS_FILE=/conf/jwks.json
-      - COOKIE_DOMAIN=${COOKIE_DOMAIN}
```
[the remark this answers is not in the document]
Comment (MSevey) [reply docker-compose.accounts.yml:35]: We might be able to close this. I'm not seeing the error anymore but we are also working on debugging other health checks.

**Comment by kwypchlo** [comment]
will be addressed in https://github.com/SkynetLabs/skynet-webportal/pull/1225

**Pull request closed without merging**